### PR TITLE
Stream file

### PR DESCRIPTION
**Pull request by Neetika23**

### CODE REVIEW

File: core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StreamRangeUnitTest {
+    @Test
```
Comment (vinipx) [inline core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java:13]: maybe let's break the line here

File: core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    public void whenRangeStreamUsingLimitSkip_thenRangePrints() {
```
Comment (vinipx) [inline core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java:14]: _thenPrintsRange()   maybe?

File: core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        List<Integer> range = numbers.stream()
+                .skip(2)
+                .limit(5)
+                .collect(Collectors.toList());
+
+        assertEquals(expectedRange, range);
+    }
+
+    @Test
+    public void whenRangeStreamUsingCollectingAndThen_thenRangePrints() {
```
Comment (vinipx) [inline core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java:27]: _thenPrintsRange()   maybe?

**Review by vinipx — CHANGES_REQUESTED** [review]

File: core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java (hunk 1 above)
Comment (Neetika23) [reply core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java:13]: done

File: core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java (hunk 2 above)
Comment (Neetika23) [reply core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java:14]: done

File: core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java (hunk 3 above)
Comment (Neetika23) [reply core-java-modules/core-java-streams/src/test/java/com/baeldung/stream/range/StreamRangeUnitTest.java:27]: done

**Review by vinipx — APPROVED** [review]

**Pull request merged**